packages/amplify-graphql-model-transformer/src/resources/custom-resource-types.d.ts

```diff
@@ -0,0 +1,125 @@
+import { DynamoDB } from 'aws-sdk';
+
+export as namespace CustomDDB;
+
+export type Input = DynamoDB.CreateTableInput;
+
+// TYPES BELOW ARE COPIED FROM CDK
```

Comment:
Would it be better to declare a dev-dependency on @types/aws-cdk ?

Reply:
I think this part is to avoid the import of the cdk module. I will try to add it as the dev dependency and see if the cdk module is bundled in the lambda zip. Marked as TODO and will address in next PR.